lib/mix/lib/mix/dep/loader.ex

```diff
@@ -301,17 +304,18 @@ defmodule Mix.Dep.Loader do
     |> elem(0)
   end
 
-  defp rebar_children(app, root_config, extra, manager) do
+  defp rebar_children(_app, root_config, manager) do
```

Comment:
We don't use `_app` here and I can't see where else we would need this, maybe we can drop this argument? 🤔